### ISSUE TITLE
Implement a proper install proceduce for the MPC code

### ISSUE THREAD

**Issue by AntoineRondelet** [clearmatics/zeth#270]
TL;DR:
The current UX for the mpc commands isn't optimal. The README (in `mpc`) assumes that the user has cloned `powersoftau` and `zeth` under the same parent directories, and has build the binaries from source (using a `release` build for PoT's binaries). Nevertheless, if binaries are distributed to users independently (or if they are moved in custom location in a docker container for instance), it is necessary to use the `--pot-path` and `--mpc-tool` flags, which may not be desirable in all circumstances (especially, when the binaries have already been moved in the PATH). Implementing a proper installation process for the mpc code, would certainly improve the UX associated to the mpc commands, by keeping the commands as simple as possible, and making sure that only minimal assumptions on the directory layout are made.

For more details, see this thread with @dtebbs https://github.com/clearmatics/zeth/pull/266#discussion_r484454593
